audioqc.rb

```diff
@@ -99,14 +99,18 @@ def CheckAudioQuality(input)
 end
 
 fileinputs.each do |fileinput|
+  fileinput = File.expand_path(fileinput)
   $file_results << fileinput
   CheckAudioQuality(fileinput)
   MediaConchScan(fileinput)
   $write_to_csv << $file_results
   $file_results = Array.new
 end
 
-CSV.open(File.expand_path("~/Desktop/audioqc-out.csv"), 'wb') do |csv|
+timestamp = Time.now.strftime('%Y-%m-%d_%H-%M-%S')
+CSV.open(File.expand_path("~/Desktop/audioqc-out_#{timestamp}.csv"), 'wb') do |csv|
+  headers = ['Filename','Levels Warnings','Phase Warnings','MediaConch Policy Compliance']
```

Comment:
Hey! Looks like we were working on this simultaneously! I added an extra column that would go between `'Levels Warnings'` and `'Phase Warnings'` that gives the actual amount of audio frames that exceeded the warning level, so I guess a header needs to be added for that. Something along the lines of `'High Level Frame Number'`